tests/test_test.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+from _pytest.capture import CaptureFixture
+from _pytest.monkeypatch import MonkeyPatch
+
+import rasa.model
+import rasa.cli.utils
+
+
+def monkeypatch_get_latest_model(tmp_path: Path, monkeypatch: MonkeyPatch):
+
+    latest_model = tmp_path / "my_test_model.tar.gz"
+
+    def mock_get_latest_model():
+        return str(latest_model)
+
+    monkeypatch.setattr(rasa.model, "get_latest_model", mock_get_latest_model)
+
+
+def test_test_core_models_in_directory_input_default(
+    capsys: CaptureFixture, tmp_path: Path, monkeypatch: MonkeyPatch
+):
+    from rasa.test import _test_core_models_in_directory_input
+
+    monkeypatch_get_latest_model(tmp_path, monkeypatch)
+
+    latest_model = Path(rasa.model.get_latest_model())
+    latest_model.touch()
+
+    # Input: default model file
+    # => Should return containing directory
+    modeldir = rasa.model.get_latest_model()
+    p = Path(modeldir)
+    new_modeldir = _test_core_models_in_directory_input(modeldir)
+    captured = capsys.readouterr()
+    assert captured.out == ""
```

Comment:
optional: a bit slicker way to check it:
```suggestion
    assert not captured.out
```